frontend/src/fixtures/courseDetailsFixtures.js

```diff
@@ -0,0 +1,20 @@
+const courseDetailsFixtures = {
+  twoCourses: [
+    {
+      quarter: "20211",
+      enrollmentCode: "ABC123",
```

Comment:
Can we make the enrollment code fixtures match the format of actual enrollment codes?  Those are 5 digit numbers.
```suggestion
      enrollmentCode: "12345",
```